exercises/all-your-base/all_your_base_test.cpp

```diff
@@ -102,50 +102,35 @@ TEST_CASE("multiple_zeros") {
 
 TEST_CASE("leading_zeros") {
     vector<unsigned int> in_digits{0, 6, 0};
-    vector<unsigned int> expected{};
+    vector<unsigned int> expected{4, 2};
     vector<unsigned int> out_digits = all_your_base::convert(7, in_digits, 10);
 
     REQUIRE(expected == out_digits);
 }
 
 TEST_CASE("first_base_is_one") {
     vector<unsigned int> in_digits{};
-    vector<unsigned int> expected{};
-    vector<unsigned int> out_digits = all_your_base::convert(1, in_digits, 10);
-
-    REQUIRE(expected == out_digits);
+    REQUIRE_THROWS_AS(all_your_base::convert(1, in_digits, 10), std::invalid_argument);
```

Comment:
Please add a blank line above this line in each of the test cases, just to match the visual style of the other tests.